tests/strct/test_structure_manipulation_functions.py

```diff
@@ -95,3 +97,59 @@ def test_add_structure_random_molecules_error():
         str(error.value)
         == "Could not add guest structure, host structure seems to be too aggregated."
     )
+
+
+def test_scale_unit_cell_uniform_scaling():
+    """Test scale_unit_cell with uniform scaling factors."""
+    structure = Structure.from_file(STRUCTURES_PATH + "MOF-5_prim.xsf")
+    scaling_factors = 1.1
+    scaled_structure = scale_unit_cell(structure, scaling_factors=scaling_factors)
+    expected_cell = np.array(structure["cell"]) * scaling_factors
```

Comment:
For the `expected_cell` I am not sure if we should have a reference structure or if the function is enough. Maybe @hdsassnick can clarify this. In general, I am not a big fan of testing a function with the same function, but in this case it could be fine. 

This thought is for all `expected_cell`

Reply:
Even if I'm not @hdsassnick, I still put in my 2 cents 😄 

I think the approach is perfectly fine. In this way, you actually test that the function is doing what you expect, since you manually implement the required step again.

Reply:
@t-reents sorry, I didn't want you to feel left out :D 

Yeah, you are correct. Also, if the structure changes (I do not know why we should change `"MOF-5_prim.xsf"` but you never know) the function would still work with another function. So, yes, it should be fine and you can ignore my comment!